torchvision/models/quantization/inception.py

```diff
@@ -128,6 +128,7 @@ def __init__(
         num_classes: int = 1000,
         aux_logits: bool = True,
         transform_input: bool = False,
+        init_weights: Optional[bool] = None,
```

Comment:
This would introduce a new API surface. Not sure if we want that. cc @datumbox

In case we don't, we can simply hardcode `init_weights=True` in the call to the superclass constructor.

Reply:
@pmeier I believe the `init_weights` and `dropout` parameters are unintentionally missed here. This issue is caused because we didn't follow the normal idiom for `Inception3`. Here is how `GoogLeNet` does it:
https://github.com/pytorch/vision/blob/a89b1957a62e2f68f001d5d60268743edbe164d8/torchvision/models/quantization/googlenet.py#L76-L79


Normally we grab args and kwargs, overwrite the things we want and pass them to the super constructor. IMO `QuantizableInception3` should do exactly the same. Instead of trying to redefine all the parameters with the same defaults, we should just do the redirection.

@ambujpawar does the way I describe the above make sense to you? Would you be able to make the change? Happy to help

Reply:
Hi, yeah it makes sense to me. I wanted to the same thing initially but was not sure if I should progress that way. 
I am happy to work on that as well.

Shall it be done as a part of this PR? Or shall I create a new issue for it?
And wouldn't it also require changes in the API surface that @pmeier mentioned above?

Reply:
We should do it here. The changes on the API surface are OK. There are no BC breakages and this is exactly how every other model does it currently (GoogleNet, MobileNets, ResNets, ShuffleNet etc). This is literally the only model that doesn't follow the idiom and that's why we got on hot water here. Someone added after the fact the parameters on the base class but forgot to do so on the Quantized version.

Reply:
Cool, I will get started with it then! Thanks for making it clear for me 👍